types/airtable/index.d.ts

```diff
@@ -1,107 +1,131 @@
-                | string
-                | number
-                | boolean
-                | Collaborator
-                | ReadonlyArray<Collaborator>
-                | ReadonlyArray<string>
-                | ReadonlyArray<Attachment>;
-        }
-
-        interface AirtableOptions {
-            apiKey?: string;
-            endpointUrl?: string;
-            apiVersion?: string;
-            allowUnauthorizedSsl?: boolean;
-            noRetryIfRateLimited?: boolean;
-            requestTimeout?: number;
-        }
-
-        interface Base {
-            (tableName: string): Table<{}>;
-        }
-
-        interface Table<TFields extends FieldSet> {
-            select(opt?: SelectOptions): Query<TFields>;
-            find(id: string): Promise<Record<TFields>>;
-            create(record: TFields, opts?: { typecast: boolean }): Promise<Record<TFields>>;
-            create(records: TFields[], opts?: { typecast: boolean }): Promise<Records<TFields>>;
-            update(...args: any[]): Promise<any>;
-            replace(...args: any[]): Promise<any>;
-            destroy(...args: any[]): Promise<any>;
-        }
-
-        interface SortParameter {
-          field: string;
-          direction?: 'asc' | 'desc';
-        }
-
-        interface SelectOptions {
-            fields?: string[];
-            filterByFormula?: string;
-            maxRecords?: number;
-            pageSize?: number;
-            sort?: SortParameter[];
-            view?: string;
-            cellFormat?: 'json' | 'string';
-            timeZone?: string;
-            userLocale?: string;
-        }
-
-        interface Query<TFields extends object> {
-            all(): Promise<Records<TFields>>;
-            firstPage(): Promise<Records<TFields>>;
-            eachPage(pageCallback: (records: Records<TFields>, next: () => void) => void): Promise<void>;
-        }
-
-        interface Record<TFields> {
-            id: string;
-            fields: TFields;
-        }
-
-        type Records<TFields> = ReadonlyArray<Record<TFields>>;
-
-        interface Attachment {
-            id: string;
-            url: string;
-            filename: string;
-            size: number;
-            type: string;
-            thumbnails?: {
-                small: Thumbnail;
-                large: Thumbnail;
-                full: Thumbnail;
-            };
-        }
-
-        interface Thumbnail {
-            url: string;
-            width: number;
-            height: number;
-        }
-
-        interface Collaborator {
-          id: string;
-          email: string;
-          name: string;
-        }
-    }
+// TypeScript Version: 3.7
+
+export type baseCreateFunctor = (tableName: string) => Table;
+
+export default class Airtable {
+    constructor(options?: AirtableOptions);
+    base(appId: string): baseCreateFunctor;
+    static base(appId: string): baseCreateFunctor;
+    static configure(options: AirtableOptions): void;
+}
+
+export class Error {
```

Comment:
Internally, this is a class called `AirtableError`. Why was this renamed to `Error` here?